src/jbi/whiteboard_tags/README.md

```diff
@@ -0,0 +1,37 @@
+When looking to add a new whiteboard tag with downstream mapping follow the format below:
+
+## Minimal Config:
+```json
+{
+    "whiteboard_tag": "wb_tag",
+    "jira_project_key": "WBTAG",
+}
+```
+
+### Caveats:
+- using the default action requires the `whiteboard_tag` and `jira_project_key` fields
+    - "action" defaults to "default_action"
+    - "enabled" defaults to true
+- there is check to confirm the `whiteboard_tag` is also the filename (`whiteboard_tag`.json)
+    - the above example is expected to exist in a file named: "wb_tag.json"
+    - the below example is expected to exist in a file named: "example.json"
```

Comment:
I modified the code locally, but the files were expected as {whiteboard_tag}.json; so that users, on a glance, can determine which whiteboard's were already in use. (Per discussion with stakeholders).

Reply:
Ok, gotcha :ok_hand: